src/app/(auth)/helpers.ts

```diff
@@ -0,0 +1,41 @@
+import { auth } from '@/auth'
+import { findUserByEmail } from '@/backend/user/user.service'
+import { Role, type User } from '@prisma/client'
+
+interface UserAuthed {
+  id: string
```

Comment:
**nit:** These are unused types, we can remove it

Reply:
done